ui/lib/utils/store.ts

```diff
@@ -45,10 +45,12 @@ export async function reloadWhoAmI() {
     store.update((s) => {
       s.whoAmI = resp.data
     })
+    return true
   } catch (ex) {
     store.update((s) => {
       s.whoAmI = undefined
     })
```

Comment:
How about throwing the error directly?

Reply:
I think using boolean as the return value may be more explicit.